{{ cookiecutter.github_repo_name }}/src/{{ cookiecutter.package_dir_name }}/app.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+
+from {{cookiecutter.package_dir_name}}.version import __version__  # noqa
```

Comment:
`# noqa` comment would be around... maybe we need to skip this file for flake8 or skip this specific error flake8 error type?